### PR TITLE
Use `alpine` as image for aws-cni's `routes-fixer`.

### PR DESCRIPTION
**Pull request by whites11**
Since latest update of aws-cni, the image does not have the tools needed to run `routes-fixer` any more.
Thus we need to run alpine

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: service/controller/resource/prepareawscniformigration/create.go [hunk 1]
```diff
@@ -109,8 +110,8 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		r.logger.Debugf(ctx, "Daemonset %q doesn't have routes-fixer container", dsName)
 		ds.Spec.Template.Spec.Containers = append(ds.Spec.Template.Spec.Containers, corev1.Container{
 			Name:    "routes-fixer",
-			Image:   ds.Spec.Template.Spec.Containers[0].Image,
-			Command: []string{"/usr/bin/bash"},
+			Image:   fmt.Sprintf("%s/giantswarm/alpine:3.17.3", r.registryDomain),
```
Comment (njuettner) [inline service/controller/resource/prepareawscniformigration/create.go:113]: Not sure if we really want to hard set the image version of alpine here. 
Is there any other way to get the version somewhere?

Comment (whites11) [reply service/controller/resource/prepareawscniformigration/create.go:113]: would you rather have this as an helm chart value?

**Review by njuettner — APPROVED** [review]
Not blocking here, we will hopefully get rid off it anyway

File: service/controller/resource/prepareawscniformigration/create.go (hunk 1 above)
Comment (njuettner) [reply service/controller/resource/prepareawscniformigration/create.go:113]: Do we need to bump it regularly, if so helm chart value might make sense?

Comment (njuettner) [reply service/controller/resource/prepareawscniformigration/create.go:113]: No worries, let's merge it.

**Pull request merged**